src/Symfony/Component/Validator/Constraints/IbanValidator.php

```diff
@@ -64,52 +143,74 @@ public function validate($value, Constraint $constraint)
             return;
         }
 
-        // ...start with a country code...
-        if (!ctype_alpha($canonicalized{0}) || !ctype_alpha($canonicalized{1})) {
+        // ...contain only digits and characters...
+        if (!ctype_alnum($canonicalized)) {
             if ($this->context instanceof ExecutionContextInterface) {
                 $this->context->buildViolation($constraint->message)
-                    ->setParameter('{{ value }}', $this->formatValue($value))
-                    ->setCode(Iban::INVALID_COUNTRY_CODE_ERROR)
-                    ->addViolation();
+                              ->setParameter('{{ value }}', $this->formatValue($value))
+                              ->setCode(Iban::INVALID_CHARACTERS_ERROR)
+                              ->addViolation();
```

Comment:
this must be reverted. Please don't change the Symfony coding standard. We use 4 spaces per indentation level